Assets/LeapMotionModules/Interaction/Examples/Basic Example/InteractionCube (1).prefab

```diff
@@ -0,0 +1,143 @@
+%YAML 1.1
```

Comment:
What is this unsightly "InteractionCube (1).prefab" sitting in your folder hierarchy?